tutormfe/plugin.py

```diff
@@ -82,20 +81,21 @@ def _add_core_mfe_apps(apps: dict[str, MFE_ATTRS_TYPE]) -> dict[str, MFE_ATTRS_T
     return apps
 
 
-@functools.lru_cache(maxsize=None)
+@tutor_hooks.lru_cache
 def get_mfes() -> dict[str, MFE_ATTRS_TYPE]:
     """
     This function is cached for performance.
     """
     return MFE_APPS.apply({})
 
 
-@tutor_hooks.Actions.PLUGIN_LOADED.add()
-def _clear_get_mfes_cache(_name: str) -> None:
+@tutor_hooks.lru_cache
+def get_plugin_slots(mfe_name: str) -> list[tuple[str, str]]:
     """
-    Don't forget to clear cache, or we'll have some strange surprises...
+    This function is cached for performance.
     """
-    get_mfes.cache_clear()
+    i: tuple[str, str, str]
```

Comment:
This annotation should not be necessary, let's get rid of it.

Reply:
Yeah, this was just a dumb attempt to get the linting to stop complaining before I got python 3.9 set up locally. 😅